Backend/app/dashboard/Lib/src/Domain/Action/Dashboard/Roles.hs

```diff
@@ -88,7 +89,7 @@ assignAccessLevel _ roleId req = do
   case mbAccessMatrixItem of
```

Comment:
here should be
`mbAccessMatrixItem <- QMatrix.findByRoleIdAndEntityAndActionType roleId req.apiEntity req.userActionType`

`QMatrix.findByRoleIdAndEntity` can be removed at all, because it make no sense now

Reply:
yes